### ISSUE TITLE
Update commission pages

### ISSUE THREAD

**Issue by Robb-Fr** [clicepfl/clic-website#155]
Correct, update or add new content to the commissions pages.

**Comment by Robb-Fr** [comment]
Would it be possible to make it a permanent issue with an associated branch (as it is the case for the news) ?

**Comment by dialexo** [comment]
Yes great idea, I advise you use `155-tom-commissions` for consistency

**Comment by s314cy** [comment]
To avoid boilerplating with "if commission.slug == ..." to display sponsors, add a "sponsors" boolean field to commission-list.json to choose whether or not to display sponsors (reuse components/sponsors/). I'll do it asap
